app/components/App.jsx

```diff
@@ -5,9 +5,9 @@ import Sidebar from './Sidebar'
 import Notifier from './Notifier'
 
 const App = ({ categories, children }) => (
-  <div role='application'>
+  <div className='con-wrapper coz-sticky'>
```

Comment:
It's not suppose to be here https://github.com/cozy/cozy-my-accounts/blob/master/app/index.ejs#L22 ?